platform/ui/src/components/Button/Button.jsx

```diff
@@ -113,13 +113,13 @@ const Button = ({
   return (
     <button
       className={classnames(
+        className,
         baseClasses,
         variantClasses[variant][color],
         roundedClasses[rounded],
         sizeClasses[size],
         fullWidthClasses[fullWidth],
-        disabledClasses[disabled],
-        className
+        disabledClasses[disabled]
```

Comment:
Not sure how much it matters, but if we have a lot of these, maybe should be alphabetic.
Order mostly matters for `className` first as override, and `baseClasses`, everything else shouldn't conflict?

Reply:
it actually the order here does not metter, I did this to try to fix label size, but didn't work and I forget to revert. the